src/Bedrock.Framework.Experimental/Protocols/Tls/Certificates/Certificate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bedrock.Framework.Experimental.Protocols.Tls.Hashs;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls.Certificates
+{
+    internal abstract class Certificate
```

Comment:
Why not X509Certificate?

Reply:
There is an implementation of this that takes in an X509Certificate that I will upload next, the code is a bit of a mess though.  In fact I would only have the version of it that is the "managed" X509Certificate version. So maybe I can flesh it out in this class directly next.

Reply:
Do we need an abstraction on top?

Reply:
Possibly not and if we need it later we can put it back, I have removed it for now (I suspect some extension methods in the future will do).